website/docs/reference/global-configs/indirect-selection.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Indirect selection"
+id: "indirect-selection"
+sidebar: "Indirect selection"
+---
+
+Use the `--indirect_selection` flag to `dbt test` or `dbt build` to configure which tests to run for the nodes you specify. You can set this as a CLI flag or an environment variable. In dbt Core, you can also configure user configurations in [YAML selectors](/reference/node-selection/yaml-selectors) or in the `config:` block of `profiles.yml`, which sets default values for all projects.
```

Comment:
I _think_ is moving to `dbt_project.yml` within https://github.com/dbt-labs/docs.getdbt.com/pull/4814

So we'll want to update to `dbt_project.yml` (after confirming that it actually works, of course).